src/Bridge/Doctrine/Orm/Enum.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace ApiPlatform\Core\Bridge\Doctrine\Orm;
+
+interface Enum
```

Comment:
We could use https://github.com/eloquent/enumeration to make it more elegant but it's another dependency.
WDYT @dunglas @alanpoulain?

Reply:
No I don't think it's a good idea. The interfaces are defined by us (we are the ones giving a contract).